### PR TITLE
add more exports so users can change db configs

### PR DESCRIPTION
**Pull request by xianny**
Running into compilation errors when trying to specify different db configs. 

This PR exports more members so they become known types when compiling the importing code. 

Example:
```
import { defaultConnectionOptions } from '@0x/coordinator-server/ts/src/default_ormconfig';

const alternateDbConfigs: ConnectionOptions = {
    ...defaultConnectionOptions,
    type: 'sqlite',
    database: 'database.sqlite_copy',
};

anotherCoordinatorServerApp = await getAppAsync(
    {
        [config.networkId]: provider,
    },
    coordinatorServerConfigs,
    alternateDbConfigs,
);
```

Build errors:
Screen Shot 2019-04-26 at 8 47 17 PM

### CODE REVIEW

File: ts/src/handlers.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ enum ExchangeMethods {
     MarketSellOrdersNoThrow = 'marketSellOrdersNoThrow',
     MarketBuyOrders = 'marketBuyOrders',
     MarketBuyOrdersNoThrow = 'marketBuyOrdersNoThrow',
-
+    MatchOrders = 'matchOrders',
```
Comment (fabioberger) [inline ts/src/handlers.ts]: Why did you add this one?

Comment (xianny) [reply ts/src/handlers.ts]: Oooops. I think this is leftover from something :/ I didn't mean to, will remove it.
